piapage/__init__.py

```
@@ -420,18 +410,17 @@ def remote_medium_url_for_id(id, is_movie):
             suffix = 'gif'
         elif is_movie:
-            return PiaPage.PHOTOJOURNAL_URL + '/animation/' + id + '.gif'
+            return Galleries.PHOTOJOURNAL_URL + '/animation/' + id + '.gif'
         else:
             suffix = 'jpg'
 
-        return PiaPage.PHOTOJOURNAL_URL + '/jpegMod/' + id + '_modest.jpg'
+        return Galleries.PHOTOJOURNAL_URL + '/jpegMod/' + id + '_modest.jpg'
 
     @staticmethod
     def local_page_url_for_id(id):
```

Ruff (0.15.9)

[warning] Missing return type annotation for staticmethod `local_page_url_for_id`

(ANN205)